.github/workflows/CI.yml

```
@@ -27,7 +27,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        os: [ubuntu-latest, macos-12]
+        os: [macos-14]
```

```suggestion
        os: [ubuntu-latest, macos-12, macos-latest]
```
This change is strictly unnecessary but I think it'd be useful to have here instead of downstream

---

Ah I had intended to just test osx-arm64 -- but would you like to get osx-arm64 added into your CI matrix?

---

up to @j-wags in this project but it makes sense to test on all three IMHO